README.md

```diff
@@ -44,6 +45,38 @@ release.
 
 An example of such `release.yml` can be found at: https://github.com/snowdrop/spring-boot-bom/blob/sb-2.3.x/release.yml
 
+### Init release repositories for a new Major.Minor
+
+Whenever a new Major.Minor release occur some of the git repositories used in the release process need some restructuring.  
+For instance, a new `sb-<major>.<minor>.x` branch must be created in the `snowdrop/spring-boot-bom` github repostory and a new
+folder in the `middleware/build-configurations` gitlab repository must also be created.
```

Comment:
Should be "Whenever a new `Major.Minor` release occurs: 
- A new `sb-<major>.<minor>.x` branch must be created under  the `snowdrop/spring-boot-bom` github repository and 
- A new folder under the `middleware/build-configurations` gitlab repository.

Reply:
Changes implemented. https://github.com/jacobdotcosta/release-manager/tree/init-release-folders#init-release-repositories-for-a-new-majorminor